mwdb/web/src/components/RichAttribute/MarkedMustache.tsx

```diff
@@ -212,8 +226,43 @@ class MarkedTokenizer extends Tokenizer {
 const mustacheWriter = new MustacheWriter();
 const markedTokenizer = new MarkedTokenizer();
 
+type stringOpFunc = (I: string) => string;
+
+const lambda = (func: stringOpFunc = _.identity) => {
+    // A factory method for custom lambdas.
+    //
+    // The inner method receives the text inside the section
+    // and the subrenderer. It then renders the value, and passes
+    // it to a user defined function.
+    //
+    // E.g.: ("{{name}}", renderer) => "John Doe" 
+    // (func is toUpperCase)        => "JOHN DOE"
+    return () => function(text: string, renderer: any): string {
+        return func(renderer(text.trim()));
+    }
+};
+
+const lambdas = fromPlugins("mustacheExtensions").reduce(
+    (prev, curr) => {
+        return { 
+            ...prev, 
+            ..._.mapValues(curr, (func: stringOpFunc) => lambda(func)) 
+        }
+    },
+{});
+
 export function renderValue(template: string, value: Object, options: Option) {
-    const markdown = mustacheWriter.render(template, value);
+    const markdown = mustacheWriter.render(
+        template, 
+        { 
+            ...value, 
+            "value": 
+            { 
+                ...(value as any).value, 
+                ...lambdas 
+            } 
+        }
+    );
```

Comment:
`value` doesn't have to be a dictionary object e.g. `List of entries` example doesn't render correctly. It's better to put lambdas in top-level view object.
```suggestion
    const markdown = mustacheWriter.render(
        template, 
        { 
            ...value,
            ...lambdas
        }
    );
```

The best solution would be a dedicated collection in MustacheContext.